### ISSUE TITLE
Missing docs: Can we update the documentation with regards to JSON credentials loading *not* from a file reference?

### ISSUE THREAD

**Issue by lirantal** [googleapis/nodejs-storage#2122]
Hi folks,

I'd like to send a pull request to update the project's README.md with instructions and examples on how to load credentials not by specifying a file path to a JSON file but rather using the `credentials` key. This isn't documented publicly anywhere and was only discovered by scouring the issue queue and a google search leading to it.

**Comment by shaffeeullah** [comment]
Hi @lirantal ,

Thanks for calling this out!! While I see why this is confusing and not well documented, I don't think the README is the right place for this information. Could you add this sample to the storage constructor file instead? https://github.com/googleapis/nodejs-storage/blob/784bd720c3392d13f39463442e7758c9a34f2d0c/src/storage.ts#L620-L652
There are a few examples there already.

Thanks again for the contribution!!

**Comment by lirantal** [comment]
Happy to extend a hand and help.
I'm not sure why the README isn't the proper place though. I would say the current README update in this PR is a good elaborate one, where-as a code example snippet on the docblock will be nice but would need to be short for brevity. Realistically, no one is going to go into that file and read it, nor shouldn't they be expected to.

I'll shoot an update to that file in a new PR but am hoping we can land this one too with the hope of easing the frustration for developers.

**Comment by lirantal** [comment]
@shaffeeullah see the docblock PR here: https://github.com/googleapis/nodejs-storage/pull/2124

**Comment by shaffeeullah** [comment]
@lirantal Thanks for making this change!! I think the README will get a little hard to parse through if we start adding code snippets directly to it. Keeping this in the docstring allows people to view it if they're looking at the [API-level documentation](https://googleapis.dev/nodejs/storage/latest/index.html). It won't be necessary for them to look at the code file directly. This could potentially be a good sample to go in the bottom section of the README. I'll discuss that option with the larger team when everyone is back after the holidays. Thanks again!
